### PR TITLE
Use window subfields in manifest for newly created windows

### LINKED ISSUE #4221: nwjs 0.13 beta3 - window.open size different from manifest
Hello,
when you try to open a new window (window.open - used also in oauth login systems, where we can't set the window size as parameters) it doesn't care about the package.json window manifest setup.

In nwjs 0.12 I see the default window size is this one specified inside the package.json.

TEST CASE:
package.json
```
{
  "name": "App",
  "version": "1.0.0",
  "main": "app.html",
  "window": {
    "show": true,
    "icon": "icon.png",
    "width": 800,
    "height": 600,
    "min_width": 800,
    "min_height": 400,
    "position": "center"
  }
}
```

call `window.open('http://www.google.it')` after launch... and you will see a 512x448 window size when I expect to be 800x600.

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/29836430-nwjs-0-13-beta3-window-open-size-different-from-manifest?utm_campaign=plugin&utm_content=tracker%2F51989&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F51989&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by ghostoy**
Fixed #4221

### CODE REVIEW

**Comment by ghostoy** [comment]
@rogerwang This patch also includes test cases and documentation. Please review.

File: src/nw_content.cc [hunk 1]
```diff
@@ -720,6 +722,10 @@ void willHandleNavigationPolicy(content::RenderView* rv,
   v8::Context::Scope cscope (v8_context);
   v8::Handle<v8::Value> element = v8::Null(isolate);
   v8::Handle<v8::Object> policy_obj = v8::Object::New(isolate);
+
+  //get and set manifest from package.json on new window
+  v8::Local<v8::Value> manifest_v8 = script_context->module_system()->RunString(v8_str("nw.App.manifest.window"), v8_str("nw_app_manifest"));
```
Comment (rogerwang) [inline src/nw_content.cc]: what if nw.App.manifest.window is undefined?

**Comment by rogerwang** [comment]
and not all window fields in the manifest should apply to new windows, e.g. show: false

**Comment by ghostoy** [comment]
@rogerwang Fixed and doced the exception of `show`. Please review.

**Comment by rogerwang** [comment]
I mean at least the 'show' field need to be excluded. Is there any other field need to be excluded as well? or we just whitelist some fields we want to migrate.

**Comment by ghostoy** [comment]
I prefer to exclude some fields and make it compatible with 0.12.

Here is the list of APIs I tested on 0.12 which behave NOT inherited from parent window:
* `show`
* `fullscreen`
* `position`
* `resizable`

And I also suggested followings should be excluded:
* `kiosk` - popups should not overlaps kiosk app

@rogerwang What do you think?

**Comment by rogerwang** [comment]
OK. Please refer to this function for nw12 behavior: https://github.com/nwjs/nw.js/blob/nw13/src/nw_shell.cc#L586

**Pull request merged**